docs/gitrepo-structure.md

```diff
@@ -172,6 +183,26 @@ targetCustomizations:
     
     After secret is created, specify the secret to `gitRepo.spec.helmSecretName`. Make sure secret is created under the same namespace with gitrepo.
 
+### Using ValuesFrom
+
+The examples below include filler `values` data for a generic Helm chart.
+These examples showcase the style and format for using `valuesFrom`.
+
+Recommended `ConfigMap` contents for the `data` field:
+
+```yaml
+values.yaml: |-
+  replication: true
+  replicas: 2
+  serviceType: NodePort
+```
+
+Recommended `Secret` contents for the `data` field (to be converted into bytes):
+
+```yaml
+replication: true\nreplicas: 2\nserviceType: NodePort
```

Comment:
I vaguely remember this not actually being the syntax. In the craziness for v2.6, I forgot if it's the case.
@bashofmann @StrongMonkey should this include raw JSON instead, no newlines, etc.?

Reply:
I just reverted the reverted commit. I'll test it tomorrow, to make sure that it's right :-)

Reply:
Thanks @bashofmann. That would be my mistake, not yours, if it was wrong. Please let me know when you get the chance!

Reply:
The examples were basically fine. I changed them a bit though to include a full ConfigMap or Secret resource to make it less ambiguous.
Also, the data values of a Secret are not "converted into bytes", but just base64 encoded. I made this easier to read by using `stringData` in the example instead.

Reply:
@bashofmann I used "converted to bytes" to not get users confused that we base64 encode their secret data. We _do_ perform a byte conversion from a Golang "string" object rather than perform the base64 encoding outselves.